### ISSUE TITLE
0kb Symlink files are causing red triangles (missing meshes) and purple textures in Skyrim LE

### ISSUE THREAD

**Issue by dzee349** [Nexus-Mods/Nexus-Mod-Manager#1159]
_Please give as much information as you can, replace/remove italicized text._

**Describe the bug** 
_Hi, first of all thank you so much for the continued updates to NMM, it is my favorite mod manager of al time.  The bug I'm experiencing is that when I install a mod via NMM, some random files from the mod will install as .symlink files, and when I load up Skyrim, the objects that are .symlink files don't show up (.e.g, if it's a .symlink mesh file, the object in game will look like a giant triangle).   How do I link the .symlink file to the actual file?  Or, is there a way to disable this .symlink feature?_

**Environment**
- NMM version: _0.80.14.0_
- Mod(s) used: _Any of them, most recently [Dwemer Automatons Glowmapped](https://www.nexusmods.com/skyrim/mods/107421)_

**To Reproduce**
 _Symlink files seem to install themselves at random_

**Additional information** 
_None_

**Comment by DuskDweller** [comment]
Some files will install as symlinks when the game's HD is different from the one where you set NMM's VirtualInstall folder on.
While this is not the optimal setup it should be working as intended (save for any weird update to the games specifically ignoring that kind of files).
If don't want to use symlink files then you need to restart from scratch and set a VirtualInstall folder on the same drive as the game.

**Comment by dzee349** [comment]
Thanks!  The weird thing is that the game is installed on the same folder as the VirtualInstall folder.  It's also only about 20 .symlinks out of the several thousand texture and mesh files I have installed for the game.

I was able to search for all .symlinks on my system using the following cmd prompt: DIR /AL /S C:\ and then manually replace all all .symlinks with the original files.  This issue can be marked as result, thanks!
